client/src/Movies/SavedList.js

```diff
@@ -1,4 +1,5 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import { Link, NavLink } from "react-router-dom";
 
 export default class SavedList extends Component {
   constructor(props) {
```

Comment:
I know you didn't wrote this constructor, but it's giving you a warning when your code is compiled. You could want to remove it.